plv8_func.cc

```diff
@@ -186,15 +178,6 @@ SubTranBlock::exit(bool success)
 		ReleaseCurrentSubTransaction();
 	else
 		RollbackAndReleaseCurrentSubTransaction();
-
-	MemoryContextSwitchTo(m_mcontext);
-	CurrentResourceOwner = m_resowner;
-
-	/*
-	 * AtEOSubXact_SPI() should not have popped any SPI context, but just
-	 * in case it did, make sure we remain connected.
-	 */
-	SPI_restore_connection();
```

Comment:
Okay. Let me know when you need help testing. I have a second Mac running
Yosemite as well.

On Saturday, February 13, 2016, Jerry Sievert <notifications@github.com>
wrote:

> In plv8_func.cc
> <https://github.com/plv8/plv8/pull/140#discussion_r52833068>:
>
> > @@ -186,15 +178,6 @@ SubTranBlock::exit(bool success)
> >  		ReleaseCurrentSubTransaction();
> >  	else
> >  		RollbackAndReleaseCurrentSubTransaction();
> > -
> > -	MemoryContextSwitchTo(m_mcontext);
> > -	CurrentResourceOwner = m_resowner;
> > -
> > -	/*
> > -	 * AtEOSubXact_SPI() should not have popped any SPI context, but just
> > -	 * in case it did, make sure we remain connected.
> > -	 */
> > -	SPI_restore_connection();
>
> ok, i'll do some quick refresh reading - in the mean time, i went ahead
> and fixed the OS X builds in a separate pull request, and have been working
> to try to get travis passing again (about 12 hours down into that rabbit
> hole right now, there were a number of issues)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/plv8/plv8/pull/140/files#r52833068>.
>